modules/core/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@bitgo/account-lib": "^1.3.0",
-    "@bitgo/statics": "^4.1.0",
+    "@bitgo/statics": "../statics",
```

Comment:
Can't point this to a local statics - this will only work when working inside the monorepo.

Reply:
yup, I'll have to remove this for a publish I figure

Reply:
should be resolved but we still need statics to be bumped as a dep or RC push